src/components/anti-everyone.ts

```diff
@@ -0,0 +1,47 @@
+import * as Discord from "discord.js";
+import { Message } from "discord.js";
+import { BotComponent } from "../bot-component.js";
+import { Wheatley } from "../wheatley.js";
+import { colors } from "../common.js";
+
+const EVERYONE_EMBED = new Discord.EmbedBuilder()
+    .setColor(colors.red)
+    .setTitle("Don't ping Everyone!")
+    .setDescription(
+        // FIXME: this message might need to be re-worded
+        'Pinging "everyone" (or "here") is not allowed in this server. ' +
```

Comment:
I'd vote for something sassy like "Did you really just try to ping 28 thousand people."

Reply:
It might be fun to actually get the exact number of people currently in the guild :)

Reply:
Ah I missed the cache, was using TCCPP.memberCount. I'll update it.